core/src/components/toolbar/toolbar.ios.vars.scss

```diff
@@ -34,7 +34,7 @@ $toolbar-ios-padding-start:                         $toolbar-ios-padding-end !de
 /// @prop - Font size of the toolbar button
 /// The minimum and maximum font sizes for a toolbar button are
 /// 100% and 135%, respectively, of their default font size
-$toolbar-ios-button-font-size:                      clamp(17px, 1.0625rem, 21px) !default;
+$toolbar-ios-button-font-size:                      dynamic-font-clamp(1, 17px, 1.24) !default;
```

Comment:
`17px * 1.24 = 21.08px` which may be causing the weird screenshot diffs from the max being `21px` previously